introduction.md

```diff
@@ -1 +1,7 @@
 Hello, I am a new student and my favorite CodeSchool path is, by far, HTML.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?